patches/tModLoader/Terraria/ModLoader/Engine/HiDefGraphicsIssues.cs

```diff
@@ -48,7 +48,7 @@ public static void Init()
 			var f_MaxTextureSize = t_ProfileCapabilities.GetField("MaxTextureSize", BindingFlags.NonPublic | BindingFlags.Instance);
 			f_MaxTextureSize.SetValue(f_ProfileCapabilities_Reach.GetValue(null), 4096);
 #endif
-#if DEBUG
+#if XNA && DEBUG
 	        mainThreadId = Thread.CurrentThread.ManagedThreadId;
 	        new Hook(typeof(Texture2D).FindMethod("ValidateCreationParameters"), new hook_ValidateCreationParameters(HookValidateCreationParameters));
```

Comment:
this is still a useful hook to have for tML development. Should figure out how to re-hook it